yarn.lock

```diff
@@ -21,24 +21,25 @@
     "@octokit/plugin-rest-endpoint-methods" "^5.13.0"
 
 "@actions/http-client@^2.0.1":
-  version "2.0.1"
-  resolved "https://registry.yarnpkg.com/@actions/http-client/-/http-client-2.0.1.tgz#873f4ca98fe32f6839462a6f046332677322f99c"
-  integrity sha512-PIXiMVtz6VvyaRsGY268qvj57hXQEpsYogYOu2nrQhlf+XCGmZstmuZBbAybUl1nQGnvS1k1eEsQ69ZoD7xlSw==
+  version "2.2.1"
```

Comment:
looks like bad branch update ? please undo these changes.

Reply:
Yea, I'm not sure why it's doing that. The versions in yarn.lock change when I `yarn install`. I haven't made any changes to package.json so I'll just revert to what's on master